### DISCUSSION TITLE
给绳网系统的1个建议。

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#168] [其他]
**建议如下：**

**支持实时在线刷新帖子**：用了几个小时，发现如果有新帖子内容发布，就必须手动重新刷新一次。并且，由于图片内容较多和众所周知的网络原因，刷新一次特别缓慢，等很久才能完全正常显示。
有那么亿点点影响观感。
![e3cae9e2d4f25ff96e7b51cf00ccbb06](https://github.com/user-attachments/assets/1a39e29b-cf6b-4b6f-a86d-6139069932f5)

**Comment by xinglanowo233** [comment]
非常赞同楼主的观点

**Comment by fierce-long** [comment]
伊莉雅好可爱！！！我还非常清楚地记得第一次邂逅伊莉雅是在当时疫情的4月08号，18点4分

**Comment by SDCOM-0415** [comment]
非常赞同👍
不过话说回来这么多的文件GitHub服务器承载量瞬间大了几倍，~~好好的折磨它们吧~~
